src/main/kotlin/no/nav/helse/FeedApi.kt

```diff
@@ -14,19 +14,20 @@ internal fun Route.feedApi(topic: String, consumer: KafkaConsumer<String, Vedtak
 
     get("/feed") {
         val maksAntall = this.context.parameters["maxAntall"]?.toInt() ?: 100
-        val sekvensNr = this.context.parameters["sekvensNr"]?.toLong()
+        val sisteLest = this.context.parameters["sistLesteSekvensId"]?.toLong()
             ?: throw IllegalArgumentException("Parameter sekvensNr cannot be empty")
 
 
-        consumer.seek(topicPartition, sekvensNr)
+        consumer.seek(topicPartition, sisteLest)
```

Comment:
fyi, man seeker til den offseten man vil lese, slik at her vil første record være den meldingen vi leste sist forrige gang. er dette intended?

Reply:
Vi får et edgecase på første melding kontra påfølgende meldinger uansett. Jeg gikk for et lite hack på linje 24. Da leser vi alltid en ekstra melding, men vi dropper den fra resultatet. Kunne evt. ha seeket til sisteLeste + 1 hvis det er safe å seeke til et offset som ikke finnes, men vet ikke om det er så mye bedre?